src/utils/CalendarUtils/Render/Render.js

```diff
@@ -10,3 +12,16 @@ export const calendarWithContributions = (container, calendar, totalContribution
   calendarContainer.innerHTML = stringifiedHTMLContent;
   calendarContainer.prepend(calendarHeader);
 };
+
+export const defaultUserCalendar = async (state) => {
+  const defaultUserJsonCalendar = await GitHub.getJsonFormattedCalendarSync(
+    state.configs.proxyServerUrl, DefaultUsers.GitHub,
+  );
+
+  const restoredDefaultUserCalendar = GitHub.restoreCalendarValues(defaultUserJsonCalendar);
```

Comment:
`restoreCalendarValues` implies some previous state. How about calling it `setEmptyCalendarValues`?